http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/common/Util.java

```diff
@@ -253,11 +253,15 @@ public static SSLConfig getSSLConfigForListener(String certPass, String keyStore
             } else if (parameter.getName()
                     .equals(Constants.SERVER_ENABLE_SESSION_CREATION)) {
                 sslConfig.setEnableSessionCreation(Boolean.parseBoolean(parameter.getValue()));
-            } else if (parameter.getName()
-                    .equals(Constants.SSL_VERIFY_CLIENT)) {
-                sslConfig.setNeedClientAuth(Boolean.parseBoolean(parameter.getValue()));
             }
         }
+        if (verifyClient != null && verifyClient.equalsIgnoreCase("require")) {
```

Comment:
can we change it to.. 

if ("require".equalsIgnoreCase(verifyClient)) { ...

Reply:
Done